libarchive/archive_write_set_format_pax.c

```diff
@@ -329,17 +329,42 @@ add_pax_attr_binary(struct archive_string *as, const char *key,
 	archive_strappend_char(as, '\n');
 }
 
+static void
+archive_write_pax_header_xattr(struct pax *pax, const char *encoded_name,
+    const void *value, size_t value_len)
+{
+	struct archive_string s;
+	char *encoded_value;
+
+	if (1) {
```

Comment:
Regarding the format: GNU tar only allows extended attributes when using --format=posix, which results in the 'POSIX ustar format', identified by "ustar" (rather than "ustar  \0"). It then writes the extended attributes with the prefix SCHILY.xattr.

https://www.gnu.org/software/tar/manual/html_chapter/tar_8.html

bsdtar gives the following options:    --format {ustar|pax|cpio|shar}  Select archive format

Should LIBARCHIVE.xattr vs. SCHILY.xattr be a choice of the format or some sort of other choice?